app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Coment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
```

Comment:
- [ ] [REQUIRED] You forgot to add `:user` association because associations between models should be set up correctly according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png). Please add the `:user` to this line 🙏🏼 

```ruby
belongs_to :user
```

Reply:
done